.github/workflows/comprehensive-testing.yml

```
@@ -349,13 +334,12 @@
       - name: Setup security test results directory
         run: mkdir -p test-results
 
-      - name: Run security tests
+      - name: Run security tests  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)